Wincenttjoi/CS3223-DBMS-Project
# Move joinalgoselector to front of command

close #14 
In addition, adds in memory references for printPlan for clearer visualization of tree

How to test:
1) Run test case 2 of DemoTwoWayJoin and DemoFourWayJoin
2) Verify that indexjoin, mergejoin and nestedjoin are successfully used for both testcases (also note that asc/dsc are not needed)

Example screenshot for memory references addition:
![image](https://user-images.githubusercontent.com/25759159/158418473-8d88a67b-4946-40c2-827d-2c7cfd650986.png)
